CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Crucible project status and open source
+
+The Crucible repo is public because it has always been our intention to
+make this open-source.  We thought it was important to explain where we're
+currently at, and manage your expectations.
+
+- We are a small company.
+
+- Our current goal is support our first generation products.
+
+- We're writing Crucible in support of that goal, not as its own thing. We're
+  all working on the products, and tool development is a side effect.
```

Comment:
This sentence doesn't transplant well, since I don't think of Crucible as "tool development".

Maybe "We're all working on the products, and block storage is an important product feature."?

Reply:
Done